doc/nrf/releases_and_maturity/migration/2.4.99-cs3_to_2.6.99-cs2/migration_guide_2.4.99-cs3_to_2.6.99-cs2_environment.rst

```diff
@@ -0,0 +1,331 @@
+.. _migration_cs3_to_2_6_99_cs2_env:
+
+Update your development environment for |NCS| v2.6.99_cs2 (for v2.4.99-cs3 users)
+#################################################################################
+
+.. contents::
+   :local:
+   :depth: 2
+
+This document describes how to update your development environment from |NCS| v2.4.99-cs3 to |NCS| v2.6.99_cs2.
+
+The main development environment changes introduced by 2.6.99-cs2 for the nRF54H20 DK are the following:
+
+* The |NCS| toolchain has been updated.
+  A bootstrap script specific to the nRF54H20 DK is now available to install and update the toolchain.
+* nRF Util has now replaced nRF Command Line Tools.
+  The bootstrap script is now able to install and update nRF Util.
+* A firmware bundle is now needed to use the nRF54H20 DK.
+  See the details in the `Installing the nRF54H20 firmware bundle`_ section below.
+* The nRF54H20 SoC lifecycle state must now be set to Root of Trust (RoT).
```

Comment:
I think that was always the case. Now we have prebuilt secdom that is signed with our keys. User will not have the same level of flexibility. User may not be able to freely change between versions.
Hi @hakonfam , does the secure manifest of secdom in this bundle have downgrade prevention enabled?

Reply:
To my knowledge, DKs are sent with LCS state set to Empty. Is that correct? @hakonfam @sveinaga @ns-tolu

Reply:
That is correct. Users needs to switch to ROT after they programmed BICRs, SDFW and SCFW.

Reply:
Downgrade prevention is enabled @pdunaj